src/main/java/RacingCarGame.java

```diff
@@ -0,0 +1,51 @@
+import java.util.Random;
+import java.util.stream.IntStream;
+
+public class RacingCarGame {
+    private RacingCar[] racingCars;
+    private InputView inputView = new InputView();
+    private ResultView resultView = new ResultView();
+    private static int MOVE_CONDITION_NUMBER = 4;
+
+    private boolean isMovable(int number) {
+        return number >= MOVE_CONDITION_NUMBER;
+    }
+
+    public static void main(String[] args) {
+        RacingCarGame game = new RacingCarGame();
+        game.inputView.closeScanner();
+    }
+    public RacingCarGame() {
+        readyCarRacing();
+        startRace();
+    }
+
+    private void readyCarRacing() {
+        inputView.receiveCarCount();
+
+        int carCount = inputView.getCarCount();
+
+        racingCars = new RacingCar[carCount];
+        for (int i = 0; i < carCount; i++) {
+            racingCars[i] = new RacingCar();
+        }
+
+        inputView.receiveMoveCount();
```

Comment:
```suggestion
        int carCount = inputView.receiveCarCount();
        int moveCount = inputView.receiveMoveCount();
        ...   
```

앞서 피드백 드린대로 View에 상태를 제거하면 위처럼 필요한 값을 바로 입력받을 수 있겠네요
이후 자동차들을 준비시키고 경기를 진행하면 되지 않을까요?

자동차를 준비시키고 경기를 진행시키는 클래스를 별도로 만들어봐도 좋을 것 같네요 😄

Reply:
클래스를 좀 더 분할하는 부분 고민해보겠습니다!